### ISSUE TITLE
What is the best code?

### ISSUE THREAD

**Issue by rush2ko** [bizstation/transactd#34]
It is a sample table.
CREATE TABLE `recommand_info` (
	`InfoSN` INT(11) NOT NULL AUTO_INCREMENT,
	`UserSN` INT(11) NOT NULL DEFAULT '0',
	PRIMARY KEY (`InfoSN`),
	UNIQUE INDEX `UserSN` (`UserSN`)
)
COLLATE='utf8_general_ci'
ENGINE=InnoDB;

I want to full record count from recommand_info table.

with activeTable:

query q;
recordset rs;
q.select(L"UserSN").reject(0xFFFF);
at.read(q, rs);
count = rs.size();

It is right?

**Comment by bizstation** [comment]
No, it is not correct. *Key value to start the searching* has not been specified.
```cpp
at.index(0).keyValue(INT_MIN).read(q, rs);
```
`InfoSN` is not unsigned. The start value must be `INT_MIN` to read from the first record.

There is a much faster way than above.
```cpp
table* tb = db->openTable(L"recommand_info", TD_OPEN_READONLY);
uint_td count= tb->recordCount(false);
tb->release();
```

thanks.

**Comment by rush2ko** [comment]
Thank you so much.
